chunky/src/java/se/llbit/chunky/renderer/projection/OmniDirectionalStereoProjector.java

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2014 Jesper Öqvist <jesper@llbit.se>
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.llbit.chunky.renderer.projection;
+
+import java.util.Random;
+
+import org.apache.commons.math3.util.FastMath;
+
+import se.llbit.math.Vector3d;
+
+/**
+ * A projector for Omni-Directional Stereo (ODS) images. x is mapped to yaw, y is mapped to
+ * pitch.
+ * @see <a href="https://developers.google.com/vr/jump/rendering-ods-content.pdf">Rendering Omni‐directional Stereo Content</a>
+ */
+public class OmniDirectionalStereoProjector implements Projector {
+	/**
+	 * The eye that this projector produces rays.
+	 */
+	private final Eye eye;
+	/**
+	 * The interpupillary distance of the viewer, in meters.
+	 */
+	private double interpupillaryDistance = 0.069;
```

Comment:
It's intentionally neither static nor final since I planned to add a setting for the IPD later, as you suggested above.